byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Block.hs

```diff
@@ -94,18 +101,27 @@ bUpdPayload b = (b ^. bBody ^. bUpdProp, b ^. bBody ^. bUpdVotes)
 bEndorsment :: Block -> (ProtVer, VKey)
 bEndorsment b = (b ^. bBody ^. bProtVer, b ^. bHeader ^. bhIssuer)
 
-
 -- | Compute the size (in words) that a block takes.
 bSize :: Block -> Natural
-bSize = fromIntegral . abstractSize acctMap
+bSize b = bHeaderSize (b ^. bHeader) + bBodySize (b ^. bBody)
+
+-- | Compute the size (in words) that a block body occupies.
+bBodySize :: BlockBody -> Natural
+bBodySize = fromIntegral . abstractSize costs
   where
-    acctMap = []
+    costs = Map.fromList [ (typeOf (undefined::Maybe UProp), 1)
+                         , (typeOf (undefined::ProtVer), 1)
+                         , (typeOf (undefined::DCert), 1)
```

Comment:
This looks OK: count one, per-each delegation certificate in the block body. @nc6: thoughts?